### PR TITLE
Add command to deployer CLI

### LINKED ISSUE #1048: Allow listing of active Airnode deployments
Documentation for https://github.com/api3dao/airnode/issues/1403

### PR DESCRIPTION
**Pull request by amarthadan**
Close #1048

@wkande I've added the command only to the deployer docker image section and the deployer command reference section. Feel free to mention it in any other sections you deem fit (quick tutorials maybe?)

### CODE REVIEW

File: docs/airnode/v0.10/grp-providers/docker/deployer-image.md [hunk 1]
```diff
@@ -142,6 +143,69 @@ api3/airnode-deployer:0.9.0 deploy --auto-remove false
...
+
+:::: tabs
+
+::: tab Linux/Mac/WSL2
+
+```sh
+docker run -it --rm \
+  -v "$(pwd):/app/config" \
+  api3/airnode-deployer:0.9.0 list
+```
```
Comment (wkande) [inline docs/airnode/v0.10/grp-providers/docker/deployer-image.md:163]: @amarthadan I see that the commands themselves are using `api3/airnode-deployer:0.9.0 list` but this has been added to Airnode `v0.10` only in the docs. Is this correct?

**Review by wkande — COMMENTED** [review]
@amarthadan Can you check the comment I made about the possible version mis-match?

File: docs/airnode/v0.10/grp-providers/docker/deployer-image.md (hunk 1 above)
Comment (amarthadan) [reply docs/airnode/v0.10/grp-providers/docker/deployer-image.md:163]: Yes, you're correct, the `list` command will be available only from `v0.10`. But the examples in the `v0.10` directory all contain `0.9.0` version. So, I've just copied it. I didn't want to break some automatic update mechanism you might have. I could have mentioned it, sorry about that.

Comment (wkande) [reply docs/airnode/v0.10/grp-providers/docker/deployer-image.md:163]: Perfect then. I will take care of the references to the reply command using `v0.10`. Thanks.

**Pull request merged**